packages/gafl-webapp-service/src/pages/multibuy/add-licence/__tests__/route.spec.js

```diff
@@ -0,0 +1,45 @@
+import { getData } from '../route.js'
+import { CONTACT_SUMMARY, LICENCE_SUMMARY } from '../../../../uri.js'
+import { createMockRequest } from '../../../../__mocks__/request.js'
+import GetDataRedirect from '../../../../handlers/get-data-redirect.js'
+jest.mock('../../../../handlers/get-data-redirect.js')
+
+describe('The add licence route .getData', () => {
```

Comment:
Call this add-licence > route. Then have a separate describe black for getData e.g. https://github.com/DEFRA/rod-licensing/blob/develop/packages/gafl-webapp-service/src/pages/licence-details/licence-to-start/__tests__/route.spec.js